etcdmain/config.go

```diff
@@ -202,6 +202,9 @@ func newConfig() *config {
 	// additional metrics
 	fs.StringVar(&cfg.Metrics, "metrics", cfg.Metrics, "Set level of detail for exported metrics, specify 'extensive' to include histogram metrics")
 
+	// auth
+	fs.StringVar(&cfg.TokenOpts, "auth-token-opts", cfg.TokenOpts, "Specify auth token specific options.")
```

Comment:
just `--auth-token` if there's not going to be a separate `--auth-token` and `--auth-token-opts` flags?